curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64cb48e36c9ad56dd7a523f4.md

```diff
@@ -0,0 +1,415 @@
+---
+id: 64cb48e36c9ad56dd7a523f4
+title: Step 100
+challengeType: 0
+dashedName: step-100
+---
+
+# --description--
+
+The next step is to create a function that will show the checkpoint message when the players reaches a checkpoint.
```

Comment:
So I think that the issue here is not with the reaches word but actual the plural use of players which is incorrect because technical there is only one player in the game not multiple. 
So I will push up a commit to resolve this